app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="ToolbarStyle" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:background">@color/teal_700</item>
+        <item name="android:title">@string/app_name</item>
+        <item name="android:titleTextColor">@color/white</item>
+    </style>
+    
+    <style name="button_main_page">
```

Comment:
the name of this styles is in cammel case like ToolbarStyle maybe this can be named ButtonMainPageStyle

Reply:
Perfect, I have a mess on my brain :P